beacon-light-client/plonky2/crates/circuit_executables/Cargo.toml

```diff
@@ -119,3 +124,15 @@ path = "bin/bls12_381_components_proofs.rs"
 [[bin]]
 name = "bls12_381"
 path = "bin/bls12_381.rs"
+
+[[bin]]
+name = "deposit_accumulator_balance_aggregator_diva_circuit_data_generation"
+path = "bin/deposit_accumulator_balance_aggregator_diva_circuit_data_generation.rs"
+
+[[bin]]
+name = "deposit_accumulator_balance_aggregator_diva"
+path = "bin/deposit_accumulator_balance_aggregator_diva.rs"
+
+[[bin]]
+name = "deposit_accumulator_balance_aggregator_final_layer"
+path = "bin/deposit_accumulator_balance_aggregator_final_layer.rs"
```

Comment:
Try using the new cached build for all the circuits, including the final layer and delete `circuit_data_generation`

Reply:
I see you are already using it, so just use it for the final layer as well and delete the generation file